tools/validate_configs/test_configs/hpc-cluster-slurm-with-startup.yaml

```diff
@@ -40,10 +40,7 @@ deployment_groups:
   - id: startup
     source: modules/scripts/startup-script
     settings:
-      runners:
-      - type: shell
-        source: "modules/startup-script/examples/install_ansible.sh"
-        destination: install_ansible.sh
+      install_ansible: true
```

Comment:
nit: It is confusing to me why we are installing ansible here at all but that pre-existed this change.